app/models/team.rb

```diff
@@ -0,0 +1,4 @@
+class Team < ApplicationRecord
+  belongs_to :user
```

Comment:
@huydangquoc is this right? Steve said the team belongs to user who creates the team. And, team has many users who are on the team. I wondering if this is similar to the friendship model? Do you need a third model to put them all together?